vendor/apis/v1/authorizenet/views.py

```diff
@@ -49,19 +50,37 @@ def renew_subscription_task(json_data):
     transaction_detail = processor.get_transaction_detail(transaction_id)
 
     if not hasattr(transaction_detail, 'subscription'):
-        return JsonResponse({})
+        return None
+    if transaction_detail.subscription.payNum.pyVal == 1:
```

Comment:
Does this value determine whether or not this is the first subscription transaction? @rhimmelbauer

Reply:
yes